### PR TITLE
feat(condo): DOMA-6695 added bank account info to payment details

### PR DESCRIPTION
**Pull request by Kekmus**
earlier on the **_settings -> paymentDetails_** page information about BillingRecepient model was displayed, changed it to BankAccount model 

Screenshot 2023-08-02 at 13 45 19

### CODE REVIEW

File: apps/condo/domains/banking/components/BankAccountInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+export const BankAccountInfo = ({ bankAccount, organizationName }) => {
+
+    return (
+        <Col span={24}>
+            <Row gutter={VERTICAL_GUTTER}>
+                <BankAccountRow
+                    bankAccountName='name'
+                    bankAccountValue={organizationName}
+                    title={true}
```
Comment (sitozzz) [inline apps/condo/domains/banking/components/BankAccountInfo.tsx:42]:
```suggestion
                    title
```

File: apps/condo/domains/banking/components/BankAccountInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+import React, { CSSProperties } from 'react'
+
+import { useIntl } from '@open-condo/next/intl'
+import { Typography } from '@open-condo/ui'
+
+const DISPLAYED_BANK_ACCOUNT = ['tin', 'routingNumber', 'number']
+
+const VERTICAL_GUTTER: [Gutter, Gutter] = [0, 24]
+
+const BankAccountRow = ({ bankAccountName, bankAccountValue, title = false }) => {
```
Comment (sitozzz) [inline apps/condo/domains/banking/components/BankAccountInfo.tsx:12]:
```suggestion
const BankAccountRow = ({ bankAccountName, bankAccountValue, isTitle = false }) => {
```

File: apps/condo/domains/banking/components/BankAccountInfo.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    const intl = useIntl()
+    const recipientTitle = intl.formatMessage({ id: 'pages.condo.settings.bankAccount.' + bankAccountName })
```
Comment (sitozzz) [inline apps/condo/domains/banking/components/BankAccountInfo.tsx:14]: IMO `bankAccountName` property naming is non-obvious

File: apps/condo/domains/banking/components/BankAccountInfo.tsx (hunk 2 above)
Comment (Kekmus) [reply apps/condo/domains/banking/components/BankAccountInfo.tsx]: done

File: apps/condo/domains/banking/components/BankAccountInfo.tsx (hunk 3 above)
Comment (Kekmus) [reply apps/condo/domains/banking/components/BankAccountInfo.tsx]: done

File: apps/condo/domains/banking/components/BankAccountInfo.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (Kekmus) [reply apps/condo/domains/banking/components/BankAccountInfo.tsx]: done

**Review by sitozzz — APPROVED** [review]

**Pull request merged**